web/src/settings_account.ts

```diff
@@ -458,26 +492,31 @@ export function set_up() {
             },
         });
 
-        if (realm.realm_password_auth_enabled !== false) {
+        if (realm.realm_password_auth_enabled) {
             // zxcvbn.js is pretty big, and is only needed on password
             // change, so load it asynchronously.
+            // TODO(evy) deal with this error:
+            // Dynamic imports are only supported when the '--module' flag is set to
+            // 'es2020', 'es2022', 'esnext', 'commonjs', 'amd', 'system', 'umd',
+            // 'node16', or 'nodenext'.
```

Comment:
I opened a thread on this: https://chat.zulip.org/#narrow/channel/6-frontend/topic/dynamic.20imports.20and.20typescript